lib/grape_entity/entity.rb

```diff
@@ -472,8 +472,13 @@ def presented
 
     # Prevent default serialization of :options or :delegator.
     def inspect
-      fields = serializable_hash.map { |k, v| "#{k}=#{v}" }
-      "#<#{self.class.name}:#{object_id} #{fields.join(' ')}>"
+      object = serializable_hash
+      if object.nil?
+        "#<#{self.class.name}:#{object_id}> nil"
```

Comment:
As I know, the value of object preferable to keep inside the `<>`, so maybe this line shouldn't be like:
```suggestion
        "#<#{self.class.name}:#{object_id} nil>"
```
wdyt? 🤔